test/e2e/operator_test.go

```diff
@@ -226,6 +212,101 @@ func TestAWSLoadBalancerControllerWithDefaultIngressClass(t *testing.T) {
 	}
 }
 
+// TestAWSLoadBalancerControllerWithCredentialsSecret tests the basic happy flow for the operator
+// using the explicitly specified credentials secret.
+func TestAWSLoadBalancerControllerWithCredentialsSecret(t *testing.T) {
+	t.Log("Creating aws load balancer controller instance with credentials secret")
+
+	// provision AWS credentials for ALBC imitating the manually created secret
```

Comment:
Comments should be written using full sentences:
```suggestion
	// Provision AWS credentials for ALBC imitating the manually created secret.
```
Lately, I've adopted the mindset that if a step in a test is worth commenting, then it is probably worth logging—and I see that you do log some other steps—so you could merge or replace the log statement above with this comment.

Reply:
Changed to logs.